server/rules.go

```diff
@@ -210,14 +211,47 @@ func (server HTTPServer) listOfDisabledClusters(writer http.ResponseWriter, requ
 	}
 }
 
-// getFeedbackAndTogglesOnRules
+// getRuleToggleMapForCluster retrieves list of disabled rules and returns a map of rule_ids
```

Comment:
technically it would be a set if you use `struct{}` instead of `true`

Reply:
@tisnik if it works, I'm planning to modify it to use the empty interface like you showed us some time ago, this was simply faster without modifying more code, is it ok with you?